docs/sources/alerting/fundamentals/notification-policies/index.md

```diff
@@ -0,0 +1,152 @@
+
+Lastly, if none of the notification policies are selected the default notification policy is used.
+
+### Example
+
+Illustrated below is an example with a relatively simple notification policy tree and some alert instances.
+
+Here's a breakdown of how these policies are selected;
+
+**Pod stuck in CrashLoop** does not have a `severity` label, so none of its child policies are matched. It does have a `team=operations` label so the first policy is matched.
+
+The `team=security` policy is not evaluated since we already found a match and `continue matching siblings` was not configured for that policy.
+
+**Disk Usage – 80%** has both a `team` and `severity` label, and matches a child policy of the operations team.
+
+**Unauthorized log entry** has a `team` label but does not match the first policy (`team=operations`) since the values are not the same, so it will continue searching and match the `team=security` policy. It does not have any child policies so the additional `severity=high` label is ignored.
+
+<!--
+@TODO insert diagram
+-->
+
+## Inheritance
+
+In addition to child policies being a useful concept for routing alert instances they also inherit properties from their parent policy. This also applies to any policies that are child policies of the default notification policy.
+
+The following properties are inherited by child policies:
+
+- Contact point
+- Grouping options
+- Timing options
+- Mute timings
+
+Each of these properties can be overwritten by an individual policy should you wish to override the inherited properties.
+
+<!--
+@TODO fact check inheritance of timings, from parent or from default policy?
+-->
+
+To inherit a contact point from the parent policy, leave it blank. To override the inherited grouping options, enable `Override grouping`. To override the inherited timing options, enable `Override general timings`.
+
+### Example
+
+The example below shows how the notification policy tree from our previous example allows the child policies of the `team=operations` to inherit its contact point.
+
+That way we can avoid having to specify the same contact point multiple times for each child policy.
+
+<!--
+@TODO insert diagram
+-->
+
+## Additional configuration options
+
+### Grouping
+
+Grouping is a new and key concept of Grafana Alerting that categorizes alert instances of similar nature into a single funnel. This allows you to properly route alert notifications during larger outages when many parts of a system fail at once causing a high number of alerts to fire simultaneously.
+
+Grouping options determine _which_ alert instances are bundled together.
+
+When an alert instance is matched to a specific notification policy it no longer has any association with its alert rule.
+
+To group alert instances by the original alert rule you can set the grouping by `alertname` and `grafana_folder` (since alert names are not unique across multiple folders).
```

Comment:
```suggestion
To group alert instances by the original alert rule, set the grouping using `alertname` and `grafana_folder` (since alert names are not unique across multiple folders).
```